addons/debugger/addon.json

```diff
@@ -75,7 +75,7 @@
       "default": false
     },
     {
-      "name": "Fill and animate graphs",
+      "name": "High detail graphs",
```

Comment:
I'm not sure if this might be interpreted as increasing the precision of graph values.

Reply:
#5995 would help with this.

Reply:
Would switching this setting around and saying "low detail graphs" work better?

Reply:
> #5995 would help with this.

Not really.

Reply:
> > #5995 would help with this.
> 
> 
> 
> Not really.

I mean, we could say turning it off improves performance.

Reply:
> I mean, we could say turning it off improves performance.

Sure, but the setting name shouldn't imply that this setting has anything to do with precision anyway

Reply:
```suggestion
      "name": "Animated graphs (may affect performance)",
```